tracee-rules/signatures/golang/code_injection.go

```diff
@@ -0,0 +1,131 @@
+package main
+
+import (
+	"fmt"
+	"regexp"
+	"strings"
+
+	"github.com/aquasecurity/tracee/tracee-rules/types"
+)
+
+type codeInjection struct {
+	cb types.SignatureHandler
+}
+
+var processMemFileRegexp *regexp.Regexp
+
+func (sig *codeInjection) Init(cb types.SignatureHandler) error {
+	sig.cb = cb
+	var err error
+	processMemFileRegexp, err = regexp.Compile(`/proc/(?:\d.+|self)/mem`)
+	if err != nil {
+		return err
+	}
+	return nil
+}
+
+func (sig *codeInjection) GetMetadata() (types.SignatureMetadata, error) {
+	return types.SignatureMetadata{
+		Name:        "Code injection",
+		Description: "Possible process injection detected during runtime",
+		Tags:        []string{"linux", "container"},
+		Properties: map[string]interface{}{
+			"Severity":     3,
+			"MITRE ATT&CK": "Defense Evasion: Process Injection",
+		},
+	}, nil
+}
+
+func (sig *codeInjection) GetSelectedEvents() ([]types.SignatureEventSelector, error) {
+	return []types.SignatureEventSelector{
+		{Source: "tracee", Name: "ptrace"},
+		{Source: "tracee", Name: "open"},
+		{Source: "tracee", Name: "openat"},
+		{Source: "tracee", Name: "execve"},
```

Comment:
execveat?

Reply:
I said the same thing to Roi when we wrote the code. I don't know why it was missing from the python signature, but we decided to just port the python sig as is, and @roikol should improve it if he think so